app/forms/journeys/additional_payments_for_teaching/claim_submission_form.rb

```diff
@@ -1,6 +1,20 @@
 module Journeys
   module AdditionalPaymentsForTeaching
     class ClaimSubmissionForm < ::ClaimSubmissionBaseForm
+      def eligible_now_and_sorted
+        eligible_now.sort_by { |e| [-e.award_amount.to_i, e.policy.short_name] }
+      end
+
+      def eligible_now
+        eligibilities.select { |e| e.status == :eligible_now }
+      end
+
+      def eligible_now_or_later
+        eligibilities.select do |e|
+          e.status == :eligible_now || e.status == :eligible_later
+        end
```

Comment:
I think eventually these methods should be on the journey eligibility checker but moving these is probably too involved at the minute